packages/demo-dapp-election/.env-sample

```diff
@@ -1,5 +1,8 @@
-DEVNET_PRIVKEY="0x your key here"
+# This is a sample .env file for use in local development.
+# Duplicate this file as .env here
 
-# this is rinkarby, can use any Arbitrum chain
-L2RPC="https://rinkeby.arbitrum.io/rpc"
+# Your Private key
+DEVNET_PRIVKEY ='0x your key here'
```

Comment:
I think these extra spaces are going to break stuffs, better do a search and replace to remove them.

Reply:
done :)